dist/angular-patternfly.js

```diff
@@ -635,6 +707,333 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', ['c3ChartDefaul
+                 <input type="checkbox" ng-model="custShowXAxis">X Axis</input>
+               </label>
+               <label class="checkbox-inline">
+                 <input type="checkbox" ng-model="custShowYAxis">Y Axis</input>
+               </label>
+             </div>
+           </form>
+         </div>
+         <div class="col-md-3">
+           <form role="form" >
+             <div class="form-group">
+               <label>Chart Height</label>
+               </br>
+               <input style="height:25px; width:60px;" type="number" ng-model="custChartHeight"></input>
+             </div>
+           </form>
+         </div>
+         <div class="col-md-3">
+               <button ng-click="addDataPoint()">Add Data Point</button>
+         </div>
+       </div>
+     </div>
+   </div>
+ </file>
+
+ <file name="script.js">
+ function ChartCtrl($scope) {
+
+     $scope.config = {
+       'chartId': 'exampleSparkline',
+       'tooltipType': 'default'
+     };
+
+    var today = new Date();
+    var dates = ['dates'];
+    for (var d = 20 - 1; d >= 0; d--) {
+        dates.push(new Date(today.getTime() - (d * 24 * 60 * 60 * 1000)));
+    }
+
+     $scope.data = {
+         'total': '100',
+         'data': ['used', '10', '20', '30', '20', '30', '10', '14', '20', '25', '68', '54', '56', '78', '56', '67', '88', '76', '65', '87', '76'],
+         'xData': dates
+     };
+
+     $scope.custShowXAxis = false;
+     $scope.custShowYAxis = false;
+     $scope.custChartHeight = 60;
+
+     $scope.addDataPoint = function () {
+       $scope.data.data.push(Math.round(Math.random() * 100));
+       $scope.data.xData.push(new Date($scope.data.xData[$scope.data.xData.length - 1].getTime() + (24 * 60 * 60 * 1000)));
+     };
+   };
+ </file>
+ </example>
+ */
+angular.module('patternfly.charts').directive('pfSparklineChart', ['c3ChartDefaults',
+  function (c3ChartDefaults) {
+    'use strict';
+    return {
+      restrict: 'A',
+      scope: {
+        config: '=',
+        chartData: '=',
+        chartHeight: '@',
+        showXAxis: '@',
+        showYAxis: '@'
+      },
+      replace: true,
+      templateUrl: 'charts/sparkline/sparkline-chart.html',
+      controller: ['$scope',
+        function ($scope) {
+
+          // Create an ID for the chart based on the chartId in the config if given
+          $scope.sparklineChartId = 'sparklineChart';
+          if ($scope.config.chartId) {
+            $scope.sparklineChartId = $scope.config.chartId + $scope.sparklineChartId;
+          }
+
+          /*
+           * Convert the config data to C3 Data
+           */
+          $scope.getSparklineData = function (chartData) {
+            return {
+              x: chartData.xData[0],
+              columns: [
+                chartData.xData,
+                chartData.data
+              ],
+              type: 'area'
+            };
+          };
+
+          $scope.sparklineTooltip = function (scope) {
```

Comment:
Seems very strange to me for scope to be passed in as an argument here.  What is the purpose of this?